sites/feedback/ajax/page-feedback-en.html

```diff
@@ -0,0 +1,74 @@
+					<legend class="col-xs-12 col-sm-7 nojs-col-sm-12 col-md-9 col-lg-8 text-center text-sm-left nojs-text-left mrgn-tp-sm pr-sm-3 nojs-pr-sm-0"><span class="field-name">Did you find what you were looking for?</span></legend>
+					<div class="col-xs-12 nojs-show">
+						<button type="submit" name="helpful" value="Yes-Oui" class="btn btn-primary" aria-describedby="why-note">Yes</button>
+					</div>
+					<div class="col-xs-12 col-sm-5 col-md-3 col-lg-4 text-center text-sm-right nojs-hide">
+						<button type="submit" name="helpful" value="Yes-Oui" class="btn btn-primary mrgn-tp-sm mrgn-bttm-sm">Yes</button>
+						<button type="button" class="btn btn-primary mrgn-tp-sm mrgn-bttm-sm mrgn-lft-sm" data-wb-doaction='[
+							{"action":"removeClass","source":".gc-pft-no","class":"nojs-show"},
+							{"action":"addClass","source":".gc-pft-btns","class":"hide"}
+						]'>No</button>
+					</div>
+				</fieldset>
+				<div class="gc-pft-no nojs-show">
+					<p id="why-note" class="nojs-show mrgn-tp-lg mrgn-bttm-md"><strong>If not, tell us why below:</strong></p>
+					<p class="nojs-hide wb-inv"><strong>Tell us why below:</strong></p>
+					<div aria-live="off">
+						<fieldset class="chkbxrdio-grp">
+							<legend><span class="field-name">What was wrong?</span></legend>
+							<div class="radio">
+								<label for="problem1"><input name="problem" id="problem1" type="radio" value="I can't find the information-Je ne peux pas trouver l'information" /> I can't <strong>find</strong> the information</label>
+							</div>
+							<div class="radio">
+								<label for="problem2"><input name="problem" id="problem2" type="radio" value="The information is hard to understand-L'information est difficile à comprendre" /> The information is hard to <strong>understand</strong></label>
+							</div>
+							<div class="radio">
+								<label for="problem3"><input name="problem" id="problem3" type="radio" value="There was an error or something didn't work-Il y avait une erreur ou quelque chose ne fonctionnait pas" /> There was an error or something <strong>didn't work</strong></label>
+							</div>
+							<div class="radio">
+								<label for="problem4"><input name="problem" id="problem4" type="radio" value="Other reason-Autre raison" /> Other reason</label>
+							</div>
+							<input type="hidden" name="problem" value="" />
+						</fieldset>
+						<div class="form-group">
+							<label for="problem6" class="mrgn-tp-sm mrgn-bttm-0"><span class="field-name">Please provide more details (maximum 300 characters)</span></label>
+							<textarea id="problem6" aria-describedby="problem6-note" name="details" class="form-control full-width" maxlength="300"></textarea>
+							<p id="problem6-note"><small>You will not receive a reply. Telephone numbers and email addresses will be removed.</small></p>
+						</div>
+						<button name="helpful" value="No-Non" class="btn btn-primary">Submit</button>
```

Comment:
```suggestion
						<button type="submit" name="helpful" value="No-Non" class="btn btn-primary">Submit</button>
```

I noticed a couple of weeks ago that the postback plugin is unable to send variables associated to submit buttons that lack a ``type="submit"`` attribute. The no questionnaire was unable to send over ``helpful=No-Non`` as a result 😢.

A quick fix for the time being would be to just tack on ``type="submit"``.

Longer term (hopefully later today), I'll try sending in a PR to fix that postback plugin quirk and revise my comments to suggest removing ``type="submit"`` across-the-board from this form.

The [missing value default for the ``button`` element is the submit state](https://html.spec.whatwg.org/multipage/form-elements.html#the-button-element:missing-value-default). So IMO there isn't really any value in explicitly-defining it.

Reply:
Gonna mark this as resolved since wet-boew/wet-boew#9534 fixes it.